packaging/installer/functions.sh

```diff
@@ -10,30 +9,15 @@
 
 setup_terminal() {
     TPUT_RESET=""
-    TPUT_BLACK=""
```

Comment:
I think these definitions should be kept for completeness, even if unused and shellcheck complains about them.